PC/pyconfig.h

```diff
@@ -355,6 +355,8 @@ Py_NO_ENABLE_SHARED to find out.  Also support MS_NO_COREDLL for b/w compat */
 #       define ALIGNOF_MAX_ALIGN_T 8
 #endif
 
+#define SIZEOF_UINTPTR_T SIZEOF_VOID_P
```

Comment:
Could we avoid these non-`Py`-namespaced configure-time constants in new code, and do  `sizeof(uintptr_t)` directly?
IMO they're only there to support compilers without `sizeof`.
`sizeof` is in C99, and even not-strictly-C99 compilers without `sizeof` seem irrelevant nowadays. (If not, then you'd need to avoid the `sizeof` in the header...)

Reply:
It's not a new constant, it already exists. If you want to avoid macros without Py_ prefix, we should have a general guideline for that.

Reply:
> I don't understand. If it already exists, why are you adding it?

PC/pyconfig.h is a file only used on Windows. It's written manually, whereas pyconfig.h is generated on Unix from pyconfig.h.in by the configure script.

In short, my change fix PC/pyconfig.h to add a missing SIZEOF_UINTPTR_T. Previously, SIZEOF_UINTPTR_T wasn't used (on Windows at least).

Reply:
Ah, I see. It's used in `selectmodule`
Still, I'd like to remove it eventually. Could you help by not adding new uses? This PR can use `sizeof(uintptr_t)` instead (which I also find more readable, especially since `sizeof` is already used nearby, but that's just opinion).

Reply:
Ok, I used sizeof(uintptr_t) instead.